css/style.css

```diff
@@ -1635,3 +1637,147 @@ li.skills {
     margin-left: 49%;
   }
 }
+
+.modal-section {
+  width: 343px;
+  height: 836px;
+  background-color: white;
+  z-index: 999;
+  display: none;
+}
+
+.modal-section .modal-container {
+  z-index: 999;
+  background-color: white;
+  position: absolute;
+}
+
+.modal-section .modal-content {
+  padding: 15px;
+}
+
+.modal-section .times {
+  color: white;
+  position: absolute;
+  font-size: 38px;
+  margin-left: 284px;
+  margin-top: 3px;
+  cursor: pointer;
+}
+
+.modal-section .modal-title {
+  width: 259px;
+  height: 135px;
+  font-weight: 600;
+  font-size: 27px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.modal-section .modal-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: -2.6rem;
+}
+
+.modal-section .modal-list li {
+  height: 14px;
+  width: 68px;
+  border-radius: 6px;
+  padding: 8px 2px;
+  background-color: #ebf0ee;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 600;
+  color: #3a4a42;
+  line-height: 16px;
+  margin: 5px;
+}
+
+.modal-section .modal-paragraph {
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+}
+
+.modal-section .btn {
+  margin: 20px 0;
+  border-radius: 4px;
+}
+
+.modal-section .btn a {
+  background-color: #36b37f;
+  color: #fff;
+  text-decoration: none;
+  line-height: 24px;
+  letter-spacing: 3%;
+  padding: 10px 25px 10px 25px;
+  border-radius: 4px;
+}
+
+.modal-section .btn a img {
+  margin-left: 5px;
+  margin-bottom: -4px;
+}
+
+#prj-img {
+  width: 100%;
+}
+
+#overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #c1c7d0;
+}
+
+@media screen and (max-width: 767px) {
+  #prj-img-1 {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #prj-img {
+    display: none;
+  }
+
+  .modal-section {
+    width: 70%;
+    background-color: white;
+    margin-left: 6rem;
+    height: 150%;
+  }
+
+  #prj-img-1 {
```

Comment:
- [ ] Please don't use ids as a selector. You could add an additional class to the element and specify it in your CSS instead of ids. ID selectors have a higher specificity than attribute selectors but using ids in CSS isn't recommended and can't be reused. CSS classes can do everything IDs can.

For more info please have a look at this [article](https://stackoverflow.com/questions/48796261/dont-use-ids-in-selectors-css-then-what-to-use-instead-of-ids#:~:text=You%20could%20add%20an%20additional,can%20do%20everything%20IDs%20can.).